src/views/httpResponseWebview.ts

```diff
@@ -351,7 +351,10 @@ ${HttpResponseWebview.formatHeaders(response.headers)}`;
     }
 
     private addUrlLinks(innerHtml: string) {
-        return innerHtml.replace(this.urlRegex, '<a href="$1" target="_blank" rel="noopener noreferrer">$1</a>');
+        return innerHtml.replace(this.urlRegex, match => {
+            const parsedLink = JSON.parse(`"${match}"`);
```

Comment:
Shall we only do this for JSON response?

Reply:
Yes! How can we know that?

Reply:
@jackbravo I think you can pass the `contentType` of response from the caller, and in this method just use the `MimeUtility.isJSON` function to check whether the response body is JSON.

Reply:
done